cyral/internal/repository/useraccount/schema_loader.go

```diff
@@ -0,0 +1,24 @@
+package useraccount
+
+import "github.com/cyralinc/terraform-provider-cyral/cyral/core"
+
+type packageSchema struct {
+}
+
+func (p *packageSchema) Name() string {
+	return "HC Vault Integration"
```

Comment:
```suggestion
	return "User Account"
```

Reply:
Thanks for catching this one! Fixed in [34c2a8d](https://github.com/cyralinc/terraform-provider-cyral/pull/474/commits/34c2a8d948c132bad8b2d953865472c57ce671f1).